webapp/app/js/config/sample.local.conf.js

```diff
@@ -3,17 +3,19 @@
  * file, you can obtain one at http://mozilla.org/MPL/2.0/. */
 
 'use strict';
+var production = "https://treeherder.mozilla.org";
+var stage = "https://treeherder.allizom.org";
+var dev = "http://treeherder-dev.allizom.org";
+var same_host = "";
```

Comment:
I might vote for 'vagrant' in that case, if we think it might result in less questions in channel?